x-pack/test/screenshot_creation/apps/response_ops_docs/observability_cases/list_view.ts

```diff
@@ -58,7 +82,19 @@ export default function ({ getService }: FtrProviderContext) {
 
     it('cases list screenshot', async () => {
       await cases.navigation.navigateToApp('observability/cases', 'cases-all-title');
-      await commonScreenshots.takeScreenshot('cases', screenshotDirectories, 1400, 1024);
+      await commonScreenshots.takeScreenshot('cases', screenshotDirectories, 1700, 1024);
+    });
+
+    it('case detail screenshot', async () => {
+      await common.navigateToUrlWithBrowserHistory('observability', `/cases/${caseIdMonitoring}`);
```

Comment:
>       await common.navigateToUrlWithBrowserHistory('observability', `/cases/${caseIdMonitoring}`);

Interstingly, I couldn't get [navigateToSingleCase](https://github.com/elastic/kibana/blob/fee63e3ea9ff5664a656d6bdad25140fa895bde9/x-pack/test/functional/services/cases/navigation.ts#L25) to work here, since `observability/cases` seemed to be deemed an invalid app.